script.js

```diff
@@ -0,0 +1,95 @@
+const toogleButton = document.getElementsByClassName("toogle-button")[0];
+const overLay = document.getElementsByClassName("overlay")[0];
+const navbarLinks = document.getElementsByClassName("navbar-links")[0];
+// const close = document.getElementsByClassName('closeMenu')[0];
+// const scrollscreen = document.body('fixed-screen')[0]
+// const menuLinks = document.querySelectorAll('.navbar-links a');
+// const openModalButtons = document.querySelectorAll('[data-modal-target]')
+// const closeModalButtons = document.querySelectorAll('[data-close-button]')
+// const menuButton = document.getElementsByClassName('menu-btn')[0]
+// const menuDesktop = document.getElementsByClassName('menu-desktop')[0]
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly remove all commented(unused) code from your codebase. because it's a good practice to keep your code clean from any unused code